library/java/net/openid/appauth/AuthorizationService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 The AppAuth for Android Authors. All Rights Reserved.
+ * Copyright 2018 The AppAuth for Android Authors. All Rights Reserved.
```

Comment:
Sorry, you don't need to change the copyright year for existing files, only for new ones. Can you revert this line?